monitoring-guidelines.md

```diff
@@ -56,5 +56,12 @@ In addition to the [OpeShift Style Guide](https://github.com/openshift/enhanceme
 1. `kubernetes_operator_part_of` label indicating the operator name. Value should be set to `kubevirt`.
 2. `kubernetes_operator_component` label indicating the value of the sub operator name.
 
+Optional labels:
+1. `priority` label indicating the alert's level of importance and the order in which it should be fixed.
+  * Valid priorities are: `high`, `medium`, or `low`.
+    Higher the priority the sooner the alert should be resolved.
```

Comment:
Maybe `The Higher the priority, the sooner`?
Other then that, /lgtm

Reply:
Done